sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs

```diff
@@ -1479,41 +1479,7 @@ public async Task UpdateCheckpointDelegatesToTheStorageManager()
                     partitionId,
                     processorClient.Identifier,
                     It.Is<CheckpointPosition>(csp =>
-                        csp.SequenceNumber == sequenceNumber),
-                    It.IsAny<CancellationToken>()),
-                Times.Once);
-
-            cancellationSource.Cancel();
-        }
-
-        /// <summary>
-        ///   Verifies functionality of the <see cref="EventProcessorClient.UpdateCheckpointAsync(string, long, long?, CancellationToken)" />
-        ///   method.
-        /// </summary>
-        ///
-        [Test]
-        public async Task PreviousUpdateCheckpointCallsTheOldUpdateCheckpoint()
```

Comment:
Yeah, I removed them because they're obsolete and because we'd have to suppress use for each.  I'm good with losing the coverage and handling any bugs as they're reported.